src/solv/cpx.F90

```diff
@@ -39,13 +69,16 @@ subroutine setup_cpcmx(self, env, solvent)
         !> TODO: SETUP solute directly from TCosmo instead of reading COSMO file
         Call read_cosmo('xtb.cosmo',self%solute,'NONE',error)
         if (allocated(error)) Call env%error(error%message, source)
+#endif
```

Comment:
One thing to do could be adding a message that a routine was called which is not available. This should of course never happen in practice but better safe than sorry ;).

In the tblite part it should be done with something like this:
```suggestion
#else
     call NotImplementedError
#endif
```

Reply:
I was actually just on it ;).